pkg/server/server.go

```diff
@@ -124,12 +131,26 @@ func WithListObjectsMaxResults(limit uint32) OpenFGAServiceV1Option {
 	}
 }
 
+// WithMaxConcurrentReadsForListObjects sets a limit on the number of datastore reads that can be in flight for a given Check call.
+// This number should be set depending on the RPS expected for Check and ListObjects APIs, the number of OpenFGA replicas running,
+// and the number of connections the datastore allows.
+// E.g. if Datastore.MaxOpenConns = 100 and assuming that each ListObjects call takes 1 second and no traffic to Check API:
+// - 1 OpenFGA replica and expected traffic of 100 RPS => set it to 1.
```

Comment:
I'm torn on this one... because i'd like people to have concrete advice on how to tune this number

Reply:
It's a non-blocker, but please fix the numbering. Right now it's 1, 1, 2 

Did you mean 1, 2, 3? Or maybe we just drop the numbers, since it's not really ordered and it's just a bulleted list.